sources/packages/web/src/components/institutions/confirmation-of-enrollment/modals/ConfirmCOEModal.vue

```diff
@@ -63,14 +64,17 @@ export default {
       try {
```

Comment:
Not part of the PR but I just noticed that Enrolment is completely misspelled above "confirmEntrollment".

Reply:
Typo has been fixed.